src/Transforms/Image.php

```diff
@@ -0,0 +1,64 @@
+<?php namespace Leean\Acf\Transforms;
+
+/**
+ * Class Image.
+ *
+ * @package Leean\Acf\Transforms
+ */
+class Image
+{
+	/**
+	 * Apply the transforms
+	 *
+	 * @param array $field The field.
+	 * @return array
+	 */
+	public static function apply( $field ) {
+		if ( ! ( 'image' === $field['type'] && 'id' === $field['return_format']) ) {
```

Comment:
I think if you reverse the comparison operation we can put inside of the if the call to 

```
self::transform_image_fields( $field ); 
```
And just have 1 return instead of 2